Sources/NIO/Bootstrap.swift

```diff
@@ -311,6 +311,11 @@ public final class ClientBootstrap {
     ///     - port: The port to connect to.
     /// - returns: An `EventLoopFuture<Channel>` to deliver the `Channel` when connected.
     public func connect(host: String, port: Int) -> EventLoopFuture<Channel> {
+        // First check if the given "host" was actual an ipaddress and if so use it directly without
+        // even try to use the resolver.
+        if let address = try? SocketAddress(ipAddress: host, port: UInt16(port)) {
```

Comment:
We could, but there's no real advantage in the breaking change at this time I don't think.

Reply:
I think it should take `Int` but I'm with @Lukasa . Let's do a follow-up PR which introduces the `Int` and deprecated the `UInt16`. I think that is worth it because when we decide to release 2.0.0 we can go through, grep for `deprecated` and remove all those. So it's like a marker in the code of what to change